packages/devtools/.eslintrc.js

```diff
@@ -1,185 +1,185 @@
 const rules = {
```

Comment:
The changes here seem unrelated. I'm guessing they're because of my comment in the other PR? If so, I meant that we should avoid having literal object keys and quoted object keys. e.g., `{ "a": 1, b: 1 }`

Reply:
This isn't related to the PR... But I ran format on this file and it changed :)
I can roll this back if its problematic to include